### DISCUSSION TITLE
Laragon and Redis on WSL

### DISCUSSION THREAD

**Discussion by robisadev** [leokhoa/laragon#491] [Q&A]
Is there a way to have Laragon start my Redis server on WSL instead of from */laragon/bin/redis/redis-*/?

**Comment by Pen-y-Fan** [comment]
I haven't set up Redis, but MySQL works the same way see my blog [How to install MySQL on WSL 2 (Ubuntu)](https://pen-y-fan.github.io/2021/08/08/How-to-install-MySQL-on-WSL-2-Ubuntu/).

Redis would need to be configured to start when wsl starts. 

To start wsl when Laragon starts add a line to the Procfile, see [How to install mailpit on Windows and integrate with Laragon](https://pen-y-fan.github.io/2023/02/23/how-to-install-mailpit-in-laragon/#autostart) for an example.
